### DISCUSSION TITLE
I'm forced to put wire:key="{{str::random()}}" for pagination to work properly

### DISCUSSION THREAD

**Discussion by yungifez** [livewire/livewire#6539] [1. Help]
After updating I noticed that Livewire does not swap the contents of a table or list or anything that was paginated without adding the wire key to force it to refresh that side of the dom

Is anyone else facing this issue?

**Comment by schonhoff** [comment]
I had a similar issue. Maybe it is related to https://github.com/livewire/livewire/discussions/6473

**Reply by yungifez** [reply]
Yeah I'll take a look at that thanks

**Comment by yungifez** [comment]
This bug has been resolved
